### PR TITLE
Added mock charts to table

### PR DESCRIPTION
**Pull request by dansterenson**

### CODE REVIEW

**Review by BenSterenson — APPROVED** [review]

File: client/src/components/charts/tableMockCharts.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export default {
+  props: ["isPositive"],
+
+  data: () => ({
+    width: 10,
+    radius: 10,
+    padding: 4,
+    lineCap: 'round',
+    gradient: gradients[0],
+    value: [0, 2, 5, 9, 5, 10, 3, 5, 0, 0, 1, 8, 2, 9, 0],
```
Comment (MPalarya) [inline client/src/components/charts/tableMockCharts.vue:33]: how about creating a random sequence?
and defining
```
values[-1] > values[0] :: green
values[-1] < values[0] :: red
values[-1] == values[0] :: black/gray

File: client/src/components/charts/tableMockCharts.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+<script>
+
+const gradients = [
+  ['#32CD32'],
+  ['#FF0000']
+]
+
+export default {
+  props: ["isPositive"],
```
Comment (MPalarya) [inline client/src/components/charts/tableMockCharts.vue:25]: I don't think this should be passed in.. rather calculated as part of this component's logic

File: client/src/components/charts/tableMockCharts.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-sparkline
+      :value="value"
+      :gradient="gradient"
+      :smooth="radius || false"
```
Comment (MPalarya) [inline client/src/components/charts/tableMockCharts.vue:5]: isn't this the same as `:smooth="radius"`?

Comment (dansterenson) [reply client/src/components/charts/tableMockCharts.vue:5]: Fixed

File: client/src/components/charts/tableMockCharts.vue (hunk 2 above)
Comment (dansterenson) [reply client/src/components/charts/tableMockCharts.vue:25]: Fixed

File: client/src/components/charts/tableChart.vue [hunk 4]
```diff
@@ -23,11 +23,18 @@
     <template v-slot:item.percent_change="{ item }">
       {{ item.percent_change.toFixed(2) + "%" }}
     </template>
+    
+    <template v-slot:item.chart="{ item }">
+      <table-mock-charts is-positive=false>{{ item }}</table-mock-charts>
```
Comment (MPalarya) [inline client/src/components/charts/tableChart.vue:28]: `is-positive`

**Review by MPalarya — CHANGES_REQUESTED** [review]

File: client/src/components/charts/tableChart.vue (hunk 4 above)
Comment (dansterenson) [reply client/src/components/charts/tableChart.vue]: Fixed

**Review by MPalarya — APPROVED** [review]

**Pull request merged**